__init__.py

```diff
@@ -314,7 +314,7 @@ def _copy(self, name=None, copy_name=None):
 
     def _edit(self, name=None, value=None):
         if not name:
-            self.q.put([Action.ask_input, _("What is the name of the password to edit?"), "", "edit"])
+            self.q.put([Action.ask_input, _("What password name do you want to edit?"), "", "edit"])
```

Comment:
Maybe just "What password do you want to edit?" here then, because this implies you're going to edit the name instead of the actual password?